.nvmrc

```diff
@@ -1 +1 @@
-14.19.0
+lts/*
```

Comment:
maybe it's better to specify version?

Reply:
thing is that we want to use always latest to make sure it is still working, for 14 was specific because we could not update

Reply:
Something is telling me that this will break on version 18 😄 But yeah, I agree that it is better to always stick with lts versions.

Reply:
Then multi-xp teams should fix 😅